### PR TITLE
chore: added enum Beast

### LINKED ISSUE #6: Create Enum Beast Type
Use the following types: Fire, Water, Earth, Electric, Dragon, Ice, Magic, Rock.

### PR DESCRIPTION
**Pull request by manassehO**
# Pull Request Overview

## 📝 Summary
Added enum Beast

### Related Issues
- Closes #6 

### Type of Change
Mark with an `x` all the checkboxes that apply (like `[x]`).
- [ ] 📝 Documentation (updates to README, docs, or comments)
- [ ] 🐛 Bug fix (non-breaking change which fixes an issue)
- [x] 👌 Enhancement (non-breaking change which adds functionality)
- [ ] 💥 Breaking change (fix or feature that would cause existing functionality to change)

### What's Changed
added enum beast

### Implementation Details
Describe how you implemented the changes.

### Technical Notes
Include any technical details that reviewers should be aware of.

## 🔧 Tests Results
Describe the tests you performed to verify your changes.

### Test Coverage
- [ ] Unit Tests
- [ ] Integration Tests
- [ ] Manual Testing

### Evidence
Provide relevant evidence of testing (screenshots, test outputs, etc.).

### Testing Notes
Include any special testing considerations or edge cases checked.

## 🔜 Next Steps
Indicate actions or improvements to be taken after this PR, if applicable.

### CODE REVIEW

File: scripts/src/models/model.cairo [hunk 1]
```diff
@@ -35,6 +35,18 @@ pub enum Direction {
     Down,
 }
 
+#[derive(Copy, Drop, Serde, IntrospectPacked, Debug)]
+pub enum Beast {
```
Comment (jimenezz22) [inline scripts/src/models/model.cairo:39]: - Enums will be added to a new path called src/types.
- Each enum has its own file. For example, for this one, it should be called beast_type.cairo.
- Don't add anything to model.cairo because this file is autogenerated by the template.

Could you please add the implementation for the enum?

- IntoBeastTypeFelt252 of Into<BeastType, felt252>
- IntoBeastTypeU8 of Into<BeastType, u8>
- IntoU8BeastType of Into<u8, BeastType>

**Review by jimenezz22 — CHANGES_REQUESTED** [review]

**Comment by manassehO** [comment]
in `IntoU8BeastType of Into<u8, BeastType>`

i added a default match to Fire in case an incorrect value is passed.

Do you want it handled differently?

**Comment by jimenezz22** [comment]
> in `IntoU8BeastType of Into<u8, BeastType>`
> 
> i added a default match to Fire in case an incorrect value is passed.
> 
> Do you want it handled differently?

Yes please handle it differently by adding an undefined type for these cases

**Comment by manassehO** [comment]
Done @jimenezz22

**Review by jimenezz22 — APPROVED** [review]
lgtm thank you!!
